Tasks/AzureTestPlanV0/SimpleTimer.ts

```diff
@@ -0,0 +1,29 @@
+import { ciDictionary } from "./ciEventLogger";
```

Comment:
Do we need our own timer logic?
Can we check how other tasks are doing it or if something is available in lib code?

Reply:
Other are doing start and end at each function block. Hence, pulled out the same and implemented the simple timer class to help with reusability